airflow/models.py

```diff
@@ -4358,13 +4358,13 @@ def is_backfill(self):
     @classmethod
     @provide_session
     def get_latest_runs(cls, session):
-        """Returns the latest running DagRun for each DAG. """
+        """Returns the latest DagRun for each DAG. """
         subquery = (
             session
             .query(
                 cls.dag_id,
                 func.max(cls.execution_date).label('execution_date'))
-            .filter(cls.state == State.RUNNING)
+            .filter(cls.state.in_(State.finished()))
```

Comment:
I don't think we should filter on finished. Even if we decide to, the documentation should match